static/docs/commands-reference/repro.md

```diff
@@ -8,7 +8,7 @@ dependencies to find only those that have to be rerun.
 
 ```usage
 usage: dvc repro [-h] [-q | -v]
-                 [-f] [-s] [-c CWD] [-m] [--dry] [-i]
+                 [-f] [-s] [-c CWD] [-m] [--dry] [-i] [-R]
                  [-p] [-P] [--ignore-build-cache] [--no-commit]
                  [targets [targets ...]]
```

Comment:
The usage I see with `dvc repro -h` is
```
usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c CWD] [-m] [--dry] [-i] [-p] [-P]
                 [-R] [--ignore-build-cache] [--no-commit] [--downstream]
                 [targets [targets ...]]
```

Reply:
But remember in the docs we keep the dvc code block lines under 70 chars (maybe 71 or 72) so they don't need horizontal scrolling when rendered in the website.

Reply:
Did you mean the alphabetical order of flags?

Also, I was curious whether dvc repro -h output comes from repository `dvc` or `dvc.org`?

Reply:
the order of flags as they are printed in the `dvc -h`

`-h` output comes from the dvc repo, not dvc.org